src/main/resources/application.properties

```diff
@@ -37,3 +37,12 @@ LW6V8BbcP5MRhd2JQSRh16nWwSQJ2BdpUZFwayEEQ6UcrMfqvA0=\
 -----END RSA PRIVATE KEY-----
 
 %dev.wildfly-bot.dry-run=true
+
+# Sending emails
+quarkus.mailer.auth-methods=DIGEST-MD5 CRAM-SHA256 CRAM-SHA1 CRAM-MD5 PLAIN LOGIN
+quarkus.mailer.host=smtp.gmail.com
```

Comment:
should this be hardcoded in properties?

Reply:
It's there so the user does not need to write those properties, if they decide to use this feature